### DISCUSSION TITLE
MPTABLE.TBL error in Noah-MP 4.0.1 setup

### DISCUSSION THREAD

**Discussion by manuelbnarvaezh** [NASA-LIS/LISF#1166] [Q&A] [answered]
Greetings - I'm running into an error message  
![MPTABLE _error](https://user-images.githubusercontent.com/101730072/181647333-f5b0e2a9-bb87-4b8b-af95-d3e4a55cb1c4.png)
Previous simulations with Noah-MP 3.6 run perfectly, but when I change de MPTABLE, GENPARM and SOILPARM tables to the new ones for Noah-MP 4.0.1 it crashes. I'm attaching the config, log and MPTABLE.TBL files here:
[narvaez_sharefiles.tar.gz](https://github.com/NASA-LIS/LISF/files/9214571/narvaez_sharefiles.tar.gz)

I appreciate any suggestions. Thank you.

**Comment by osbornesci** [comment, marked as answer]
Hi Manuel and all: Manuel is using almost all of the same files we're using at JHU in order to run this LIS simulation on a different university server in Ecuador. The only files that differ are the noahmp401 "MPTABLE.TBL" and other " .TBL" files, and possibly the input greenness vegetation fraction files (minimum and maximum GVF). I am sending Manuel these files today to test whether or not they help move closer to a solution. Thanks.

**Reply by osbornesci** [reply]
Hi Manuel,

On second look at the ldt.config, our GVF settings are the same. Neither of them require input files for "Greenness maximum map" or "Greenness minimum map" since the "Calculate min-max greenness fraction" is set to ".true". If we had instead set it to ".false", then we'd need user-loaded greenness min and max maps. 

Below, I'm uploading the materials that are working for running LIS with Noah-MP v4.0 on Discover. I'm also uploading all the noahmp401 ".TBL" files including the "MPTABLE.TBL" which you can try with your existing setup (since you mentioned the version you used was from the NCAR repository which differs from the one we've used at JHU and on Discover). 

When using the files below on Discover, all Noah-MP features appear to run successfully in this case except for the section that outputs HyMAP2 variables, so you can deactivate HyMAP (or suppress HyMAP output) until this error is resolved. Please refer to this reported HyMAP output error and ongoing troubleshooting efforts here:  https://github.com/NASA-LIS/LISF/discussions/1109.

[osborne_sharefiles.zip](https://github.com/NASA-LIS/LISF/files/9292877/osborne_sharefiles.zip)

[noahmp401_parms.zip](https://github.com/NASA-LIS/LISF/files/9293251/noahmp401_parms.zip)

I hope this helps,
Tashiana
